build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -281,8 +281,8 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_REGIONx_START="0x20200000"
-CONFIG_RAM_REGIONx_SIZE="524288"
+CONFIG_RAM_REGIONx_START="0x20200000,0x80200000"
+CONFIG_RAM_REGIONx_SIZE="524288,31457280"
```

Comment:
KIndly remove CONFIG_MM_PARTITION_HEAP from defconfig as this feature was added due to removal of partition manager module

Reply:
It was disabled by default, have removed now from defconfig